packages/api-extractor-model/src/model/ModelReferenceResolver.ts

```diff
@@ -113,11 +113,19 @@ export class ModelReferenceResolver {
 			const memberSelector: DocMemberSelector | undefined = memberReference.selector;
 			if (memberSelector === undefined) {
 				if (foundMembers.length > 1) {
-					result.errorMessage = `The member reference ${JSON.stringify(identifier)} was ambiguous`;
-					return result;
+					const foundClass: ApiItem | undefined = foundMembers.find((member) => member.kind === ApiItemKind.Class);
```

Comment:
How come the type is explicitly defined here? It is already `ApiItem | undefined`, no?

Reply:
Convention in api-extractor, they explicitly define all their variable‘s types… though if I was to fully follow their convention I would omit the `| undefined` but that was too cursed even for my taste.